From 3387f440fdc781acda9ffb9e0edcd9ddc7d8c7ec Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Thu, 15 Dec 2022 15:54:50 -0500
Subject: [PATCH 31/37] Fixup of c50777 -- original commit only cares about R
 vendors, but not about older ones. Apply that on older ones as well

---
 .../core/java/com/android/server/biometrics/AuthService.java    | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/services/core/java/com/android/server/biometrics/AuthService.java b/services/core/java/com/android/server/biometrics/AuthService.java
index a121579f40d8..bc741a5e2396 100644
--- a/services/core/java/com/android/server/biometrics/AuthService.java
+++ b/services/core/java/com/android/server/biometrics/AuthService.java
@@ -653,7 +653,7 @@ public class AuthService extends SystemService {
             final int firstApiLevel = SystemProperties.getInt(SYSPROP_FIRST_API_LEVEL, 0);
             final int apiLevel = SystemProperties.getInt(SYSPROP_API_LEVEL, firstApiLevel);
             String[] configStrings = mInjector.getConfiguration(getContext());
-            if (configStrings.length == 0 && apiLevel == Build.VERSION_CODES.R) {
+            if (configStrings.length == 0 && apiLevel <= Build.VERSION_CODES.R) {
                 // For backwards compatibility with R where biometrics could work without being
                 // configured in config_biometric_sensors. In the absence of a vendor provided
                 // configuration, we assume the weakest biometric strength (i.e. convenience).
-- 
2.34.1

